.github/workflows/data-design.yml

```diff
@@ -0,0 +1,59 @@
+name: "[DevX Security] Google Chats PR Announcer"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Every morning at 9:30AM UTC, Mondays to Fridays
+    # See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
+    - cron: "30 10 * * MON-FRI"
+
+jobs:
+  # See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-defining-outputs-for-a-job
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      repos: ${{ steps.main.outputs.repos }}
+    steps:
+      - id: main
+        # See https://github.com/orgs/guardian/teams/devx-security/repositories
+        run: |
+          REPOS=(
+            data-platform-models
+            ophan-data-lake
+            gcp-iac-terraform
+          )
+
+          RESULT=""
+          for repo in "${REPOS[@]}"; do
+            RESULT="$RESULT,guardian/$repo"
+          done
+
+          # remove leading ,
+          RESULT="${RESULT:1}"
+          echo "repos=$RESULT" >> $GITHUB_OUTPUT
+  prnouncer:
+    runs-on: ubuntu-latest
+    needs: setup
+    steps:
+      - name: Use GitHub App Token
+        uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          # These values are for the GitHub App "Gu PRnouncer config"
+          # See https://github.com/organizations/guardian/settings/apps/gu-prnouncer-config (only accessible by GitHub owners)
+          app-id: ${{ secrets.APP_ID }}
+          private-key: ${{ secrets.PRIVATE_KEY }}
+          owner: guardian
+
+      - uses: guardian/actions-prnouncer@main
+        with:
+          google-webhook-url: ${{ secrets.DATA_DESIGN_GOOGLE_WEBHOOK_URL }}
+          github-repositories: ${{ needs.setup.outputs.repos }}
+          github-token: ${{ steps.app-token.outputs.token }} # A GitHub app token generated by the previous step
+
+          # By default, guardian/actions-prnouncer doesn't report PRs from Dependabot. Include them here.
+          github-ignored-users: ""
```

Comment:
minor: shouldn't think this is needed when `github-announced-users` is provided, though I've not checked the source code.